RadarSDK/RadarLogger.m

```diff
@@ -7,6 +7,8 @@
 
 #import "RadarLogger.h"
 
+#import "Radar.h"
```

Comment:
ditto here - this import might be redundant to `Radar+Internal.h`, which defines the `sendLog` method